package-res/reportviewer/messages/messages.properties

```diff
@@ -21,4 +21,10 @@ SessionExpired=Session Expired
 SessionExpiredComment=Your session has expired. Please reload this page once you have logged in again.
 
 ErrorParsingParamXmlMessage=Error parsing parameter information.
-FatalErrorTitle=Fatal Error
\ No newline at end of file
+FatalErrorTitle=Fatal Error
+
+# Async processing report Activity name
+async_computing_layout_title=Computing layout
```

Comment:
For these key names, please follow the CamelCase convention the other keys use. Consistency is a good thing.